### PR TITLE
6 AC are completed

### PR DESCRIPTION
**Pull request by kesezgin**
Please review and approve my pull request

### CODE REVIEW

File: src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java [hunk 1]
```diff
@@ -106,6 +110,157 @@ public void user_sees_a_new_customer_is_created_with_name(String string) {
...
+    }
+
+
+
+
+    @When("User enters {string} into Search Box on the top right of the page and press Enter")
+    public void user_enters_into_search_box_on_the_top_right_of_the_page_and_press_enter(String string) {
+        salesFunctionalityPages.SearchBox_Customer.sendKeys(string, Keys.ENTER);
+        try {
+            Thread.sleep(5000);
```
Comment (nevzattalhaozcan) [inline src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java:222]: You can use the sleep() method from BrowserUtils

File: src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java [hunk 2]
```diff
@@ -106,6 +110,157 @@ public void user_sees_a_new_customer_is_created_with_name(String string) {
...
+    //    salesFunctionalityPages.ActionBtn.click();
+    //    BrowserUtils.waitForElement((salesFunctionalityPages.ActionDeleteBtn));
+    //    salesFunctionalityPages.ActionDeleteBtn.click();
+    //    BrowserUtils.waitForElement(salesFunctionalityPages.ActionDeleteOkBtn);
+    //    salesFunctionalityPages.ActionDeleteOkBtn.click();
+    //}
+
+    //AC7
+    @Then("User sees {string} message under full profile")
+    public void userSeesMessageUnderFullProfile(String arg0) {
```
Comment (nevzattalhaozcan) [inline src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java:249]: You should give a meaningful name for the variable inside the parameter

File: src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java [hunk 3]
```diff
@@ -106,6 +110,157 @@ public void user_sees_a_new_customer_is_created_with_name(String string) {
...
+        try {
+            Thread.sleep(5000);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+
+    }
+
+    @Then("User sees the created Customer with {string} name")
+    public void userSeesTheCreatedCustomerWithName(String arg0) {
```
Comment (nevzattalhaozcan) [inline src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java:230]: You should give a meaningful name for the variable inside the parameter

File: src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java [hunk 4]
```diff
@@ -106,6 +110,157 @@ public void user_sees_a_new_customer_is_created_with_name(String string) {
...
+    }
+
+    @And("User enter {string} into Position Box on New Customer Pop Up")
+    public void userEnterIntoPositionBoxOnNewCustomerPopUp(String arg0) {
+        salesFunctionalityPages.jobPositionInfo.click();
+        salesFunctionalityPages.jobPositionInfo.sendKeys(arg0);
+    }
+
+    @And("User enter {string} into e-mail Box on New Customer Pop Up")
+    public void userEnterIntoEMailBoxOnNewCustomerPopUp(String arg0) {
```
Comment (nevzattalhaozcan) [inline src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java:183]: You should give a meaningful name for the variable inside the parameter

File: src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java [hunk 5]
```diff
@@ -106,6 +110,157 @@ public void user_sees_a_new_customer_is_created_with_name(String string) {
...
+
+    @And("User enter {string} into Street Box on New Customer Pop Up")
+    public void userEnterIntoStreetBoxOnNewCustomerPopUp(String arg0) {
+        BrowserUtils.waitForElement(salesFunctionalityPages.addressInfo);
+        salesFunctionalityPages.addressInfo.click();
+        salesFunctionalityPages.addressInfo.sendKeys(arg0);
+    }
+
+    @And("User enter {string} into Position Box on New Customer Pop Up")
+    public void userEnterIntoPositionBoxOnNewCustomerPopUp(String arg0) {
```
Comment (nevzattalhaozcan) [inline src/test/java/com/fidexio/step_definitions/SalesFunctionality_StepDefinitions.java:177]: You should give a meaningful name for the variable inside the parameter

**Comment by IrfanAzim34** [comment]
What a nice recommendations

Nevzat Talha Özcan ***@***.***>, 29 Mar 2023 Çar, 02:16
tarihinde şunu yazdı:

**Review by nevzattalhaozcan — APPROVED** [review]
I approve this but please read the comments I put for some parts of your code

**Comment by IrfanAzim34** [comment]
What a responsibility  :)))

Nevzat Talha Özcan ***@***.***>, 29 Mar 2023 Çar, 02:20
tarihinde şunu yazdı:

**Pull request merged**